angular-material/angular-material-tests.ts

```diff
@@ -4,7 +4,8 @@ var myApp = angular.module('testModule', ['ngMaterial']);
 
 myApp.config((
     $mdThemingProvider: ng.material.IThemingProvider,
-    $mdIconProvider: ng.material.IIconProvider) => {
+    $mdIconProvider: ng.material.IIconProvider,
+	$mdProgressCircularProvider: ng.material.IProgressCircularProvider) => {
```

Comment:
Sorry @andy-ms , i used `git rebase --onto master angular-material` but now i see other commits (master branch?) in my pull request and it's not clear. Where did I go wrong?